### PR TITLE
heatmap gui

### PR DESCRIPTION
**Pull request by amnona**

### CODE REVIEW

File: calour/filtering.py [hunk 1]
```diff
@@ -128,7 +129,7 @@ def _sum_abundance(x, cutoff=10):
 
     '''
     logger.debug('')
-    return sum(x) >= cutoff
+    return x.sum() >= cutoff
```
Comment (RNAer) [inline calour/filtering.py:131]: why the change? the previous version is more generic, working for both np array and other iterable.

File: calour/gui/plotgui_jupyter.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+from matplotlib.figure import Figure
+
+from ipywidgets import *
+from IPython.display import display, clear_output
+
+from calour.bactdb import BactDB
+from calour.gui.plotgui import PlotGUI
+
+
+# app_ref=set()
```
Comment (RNAer) [inline calour/gui/plotgui_jupyter.py]: delete it?

File: calour/heatmap.py [hunk 3]
```diff
@@ -212,7 +60,11 @@ def plot(exp, xfield=None, feature_field='taxonomy', max_features=40, logit=True
     title : None or str (optional)
```
Comment (RNAer) [inline calour/heatmap.py:60]: `title` is not used. Default should be `''` instead of `None`?

File: calour/filtering.py (hunk 1 above)
Comment (amnona) [reply calour/filtering.py:131]: better maybe to be compatible with sparse and dense?

File: calour/gui/plotgui_jupyter.py (hunk 2 above)
Comment (amnona) [reply calour/gui/plotgui_jupyter.py]: removed

File: calour/heatmap.py (hunk 3 above)
Comment (amnona) [reply calour/heatmap.py:60]: ok
